src/biogeochem/CNPhenologyMod_cut_newFlux_newCutParam_N.F90

```diff
@@ -0,0 +1,3060 @@
+                  ! this code was originally handled with call cn_storage_to_xfer(p)
+                  ! inlined during vectorization
+
+                  ! set carbon fluxes for shifting storage pools to transfer pools
+                  leafc_storage_to_xfer(p)  = fstor2tran * leafc_storage(p)/dt
+                  frootc_storage_to_xfer(p) = fstor2tran * frootc_storage(p)/dt
+                  if (woody(ivt(p)) == 1.0_r8) then
+                     livestemc_storage_to_xfer(p)  = fstor2tran * livestemc_storage(p)/dt
+                     deadstemc_storage_to_xfer(p)  = fstor2tran * deadstemc_storage(p)/dt
+                     livecrootc_storage_to_xfer(p) = fstor2tran * livecrootc_storage(p)/dt
+                     deadcrootc_storage_to_xfer(p) = fstor2tran * deadcrootc_storage(p)/dt
+                     gresp_storage_to_xfer(p)      = fstor2tran * gresp_storage(p)/dt
+                  end if
+
+                  ! set nitrogen fluxes for shifting storage pools to transfer pools
+                  leafn_storage_to_xfer(p)  = fstor2tran * leafn_storage(p)/dt
+                  frootn_storage_to_xfer(p) = fstor2tran * frootn_storage(p)/dt
+                  if (woody(ivt(p)) == 1.0_r8) then
+                     livestemn_storage_to_xfer(p)  = fstor2tran * livestemn_storage(p)/dt
+                     deadstemn_storage_to_xfer(p)  = fstor2tran * deadstemn_storage(p)/dt
+                     livecrootn_storage_to_xfer(p) = fstor2tran * livecrootn_storage(p)/dt
+                     deadcrootn_storage_to_xfer(p) = fstor2tran * deadcrootn_storage(p)/dt
+                  end if
+               end if
+
+               ! test for switching from growth period to offset period
+            else if (offset_flag(p) == 0.0_r8) then
+               if (use_cndv) then
+                  ! If days_active > 355, then remove patch in
+                  ! CNDVEstablishment at the end of the year.
+                  ! days_active > 355 is a symptom of seasonal decid. patches occurring in
+                  ! gridcells where dayl never drops below crit_dayl.
+                  ! This results in TLAI>1e4 in a few gridcells.
+                  days_active(p) = days_active(p) + fracday
+                  if (days_active(p) > 355._r8) pftmayexist(p) = .false.
+               end if
+
+               ! only begin to test for offset daylength once past the summer sol
+               if (ws_flag == 0._r8 .and. dayl(g) < crit_dayl) then
+                  offset_flag(p) = 1._r8
+                  offset_counter(p) = ndays_off * secspday
+                  prev_leafc_to_litter(p) = 0._r8
+                  prev_frootc_to_litter(p) = 0._r8
+               end if
+            end if
+
+         end if ! end if seasonal deciduous
+
+      end do ! end of patch loop
+
+    end associate
+ 
+  end subroutine CNSeasonDecidPhenology
+
+  !------------------------------------------------------
```

Comment:
I see that you transferred leaf litter (above) to the biomass pool. Were you also planning to transfer stem litter (here)?